binnacle/lib/models/provider_data.dart

```diff
@@ -1,18 +1,20 @@
-
+import '../enums.dart';
```

Comment:
I think the name should be more descriptive, even if there's only one kind of enum present in the file. We might use enums elsewhere, something like ProviderTypeEnums would do.

Reply:
So my thought was just to have a file that just holds all the enums